students/11th/hyojeung/style/login.css

```
@@ -0,0 +1,73 @@
+/* mainBox */
+.warp{
+  width: 1200px;
+  height: 100%;
+  margin: 0 auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;}
+
+.mainBox{
+  width: 350px;
+  height: 379px;
+  border: 1px solid #e6e6e6;
+  border-radius: 1px;
+  background: #fff;
+  margin: 0 auto;}
+
+.logo{
+  text-align: center;
+  margin: 35px auto 25px;}
+
+.loginBox{
+  width: 350px;
+  height: 170px;
+  display: flex;
+  text-align: center;
+}
+.loginFom{
+  justify-content: center;
```

```suggestion
}

.loginForm {
  display: flex;
  justify-content: center;
```
1. 요소간 줄바꿈 해주세요. 가독성은 매우 중요합니다.
2. 오타 수정해주세요.
3. display flex 없이 flex 관련 속성이 쓰이고 있습니다. 확인해주세요.

---

1. 요소줄바꿈 수정하였습니다.
2. loginForm로 오타수정 완료하였습니다.
3. display: flex; 수정하였습니다.